ZDNA-prediction.local.2.ipynb

```diff
@@ -0,0 +1,459 @@
+    "\n",
+    "    prediction_result_formatter_bed_file = PredictionResultFormatterBedFile()\n",
+    "    \n",
+    "    prediction_runner = PredictionRunner()\n",
+    "\n",
+    "    for prediction_result in prediction_runner.run(prediction_inputs, progress_bar=tqdm):\n",
+    "        bed_file_name_seq = prediction_result_formatter_bed_file.file_name(prediction_result)\n",
+    "        \n",
+    "        print(bed_file_name_seq)\n",
+    "        #print(bed_file_name)\n",
+    "\n",
+    "        bed_file_seq_handler = open(os.path.join(output_path, bed_file_name_seq), 'w')\n",
+    "        \n",
+    "        #print('track name=\"{name}\" priority=1'.format(name=model_params_as_string))\n",
+    "        #print('track name=\"{name}\" priority=2'.format(name=seq_name))\n",
+    "        for line in prediction_result_formatter_bed_file.format(prediction_result):\n",
+    "            #print(line)\n",
+    "            bed_file_seq_handler.write(\"%s\\n\" % line)\n",
+    "\n",
+    "        bed_file_seq_handler.close()\n",
+    "        #print()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "do_predictions_button = widgets.Button(\n",
+    "    description='Run prediction',\n",
+    "    icon='chart-line',\n",
+    ")\n",
+    "do_predictions_button.on_click(do_predictions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run\n",
+    "\n",
+    "Start predicting features of fasta file inputs in 4 steps.\n",
+    "\n",
+    "## Usage\n",
+    "\n",
+    "### Prepare\n",
+    "\n",
+    "Preparing the environment only needs to be done once everytime when starting JupyterLab or freshly opening the notebook thereafter.\n",
+    "\n",
+    "### Select model and parameters\n",
+    "\n",
+    "After changing the model or the parameters, press the \"Load model\"-Button.\n",
+    "\n",
+    "This will create required directories, download required files and move the model file into the relevant directory. Files that have been downloaded already, will not be downloaded again.\n",
+    "\n",
+    "### Run\n",
+    "\n",
+    "After the predictions have been made, new files will be created in the directory `output`.\n",
+    "\n",
+    "The following types of files will be created:\n",
+    "\n",
+    "- `.txt`-Files will contain the textual representation as seen in the output of the notebook for all input files\n",
+    "- Several different `.bed`-Files containing the found features will be created for each input file based on the selected sequence variations\n",
+    "\n",
+    "  They can be used to import found features into other software.\n",
+    "  \n",
+    "  - `.normal.bed` contains features found for the original input fasta file\n",
+    "  - `.rev-comp.bed` contains features found for the reverse-complement\n",
+    "  - `.bed` contains features found in both the normal and the reverse-complement\n"
```

Comment:
currently no `.bed`-file is generated, so probably should remove this line if not re-implemented

Reply:
combined bed file output has been reimplemented with https://github.com/AnthyG/Z-DNABERT/pull/7/commits/93f3786bd44f5356d8b259d13075691dba32ff69